volume/local/local_unix.go

```diff
@@ -71,6 +69,29 @@ func (v *localVolume) mount() error {
 	if v.opts.MountDevice == "" {
 		return fmt.Errorf("missing device in volume options")
 	}
-	err := mount.Mount(v.opts.MountDevice, v.path, v.opts.MountType, v.opts.MountOpts)
+	r1, _ := regexp.Compile(`\baddr=*`)
```

Comment:
Yes, I suggested doing that in https://github.com/docker/docker/pull/27329#discussion_r83539559, perhaps we should skip the regex (since `getAddress` is checking for it anyway to verify it's not empty)